app/assets/javascripts/Components/notes_table.jsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import {render} from 'react-dom';
+
+import ReactTable from 'react-table';
+import {admins_path, all_notes_notes_path, edit_note_path} from "../../../javascript/routes";
+
+class NotesTable extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      notes: [],
+      columns: {},
+      loading: true
+    };
+  }
+
+  componentDidMount() {
+    this.fetchData();
+  }
+
+  fetchData = () => {
+    $.get({
+      url: Routes.all_notes_notes_path()
+    }).then(res => {
+      this.setState({
+        notes: res['notes_data'],
+        loading: false,
+        columns: res['column_headers']
+      });
+    });
+  };
+
+  buttons(modifiable, id) {
+    if(modifiable) {
+      return(
+        <div>
+          <div>
+            <a href={Routes.edit_note_path(id)}>
+              <button> {I18n.t('edit')} </button>
+            </a>
+          </div>
+
+          <div>
+            <a href={"notes/" + id} data-method="delete" data-confirm={I18n.t('notes.delete.link_confirm')}>
```

Comment:
Instead of `"notes/" + id`, you can use `Routes.note_path(id)`. Also see above comment about button + styling.